mmengine/optim/scheduler/param_scheduler.py

```diff
@@ -610,7 +610,10 @@ class CosineAnnealingParamScheduler(_ParamScheduler):
             epochs. Defaults to True.
         verbose (bool): Whether to print the value for each update.
             Defaults to False.
-
+        eta_min_ratio (float, optional): The ratio of the minimum parameter
```

Comment:
Also need to update the doc string of CosineAnnealingLR and CosineAnnealingMomentum.